src/config/dark-sites.config

```diff
@@ -171,6 +171,7 @@ cyberdrop.me
 cyberpunk.net
 cybersole.io
 cytu.be
+dahliaos.io
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/127750098-f0d89f7c-736d-4cb8-95fd-985765abf8b7.png)

Reply:
mate its responsive to your theme :|

Reply:
read CONTRIBUTING.md

Reply:
Yeah, that's not allowed. See [contributing.md](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#adding-a-website-that-is-already-dark)

Reply:
![image](https://user-images.githubusercontent.com/42787085/127750181-1f6d0d93-c311-4a65-b4a9-db6b707d4105.png)

Reply:
so im just gonna remove that one

Reply:
I'm not sure why the downvote. But let me explain, why it's required to be added to the list.

People like me that somehow have light theme preference because the browser and any linux desktop manager don't like communicating with each other will go to such site and are blinded once we visit the site. And I don't like to get blinded, nor does my rubber ducky.

Reply:
> I'm not sure why the downvote. But let me explain, why it's required to be added to the list.
> 
> People like me that somehow have light theme preference because the browser and any linux desktop manager don't like communicating with each other will go to such site and are blinded once we visit the site. And I don't like to get blinded, nor does my rubber ducky.

i understand, I took away my downvote because I was being pretty stuck up, with GitHubs terrible UX it must've not updated.

Bless your eyes and your rubber ducky ❤️